### ISSUE TITLE
Login still redirect to login page with discord

### ISSUE THREAD

**Issue by akashchow** [Snazzah/stub#71]
Hi all, Good Morning,
This is writing from a self taught dev. After several attempt failed to login with discord. Redirected to blank home page: My .Env
![test](https://user-images.githubusercontent.com/33022033/198820543-3cc595bb-879b-427d-86ec-b34ab9606357.png)

There some discord app oAuth2 issue: do I need to declare redirect url as http://localhost:3000/api/auth/callback/discord and what is the scope to select like email, guild, identity etc. 

Email login send email but after signup redirect to login page again. not work.

Help required. Thanks in advance for your supports.

**Comment by Snazzah** [comment]
> There some discord app oAuth2 issue: do I need to declare redirect url as http://localhost:3000/api/auth/callback/discord and what is the scope to select like email, guild, identity etc.
Yes, you don't need to specify any scope on the app dashboard, the app will handle that itself.

Should also make sure `APP_HOSTNAME` is just `localhost:3000` without the protocol.

**Comment by vishnusharmax** [comment]
After implementing google auth and discord auth still after login redirected to login page. Any solution for this issue ?
